README.md

```diff
@@ -22,6 +22,17 @@ Once you've done some hacking and you'd like to have your work merged, you'll ne
 * libnss-devel
 * libexpat-devel
 
+### CentOS/Red Hat
```

Comment:
Can you change this to a shell invocation that can be copy-pasted to install all the packages? Something like `yum install ...`.